### PR TITLE
[Backend] Add updUser function

### PR DESCRIPTION
**Pull request by aobolensk**

### CODE REVIEW

File: webforces/server/mongodbworker.py [hunk 1]
```diff
@@ -101,6 +101,28 @@ def addUser(self, user) -> Tuple[DBStatus, User]:
...
+            if user_d is None:
+                logger.error("This user does not exist")
+                return DBStatus.s_data_issue
+            user_d = users_collection.update_one(
+                {"user_id": user.user_id}, {"$set": {
+                    "first_name": user.first_name,
+                    "second_name": user.second_name,
+                    "middle_name": user.middle_name,
+                }})
+            if user_d is None:
```
Comment (megarungle) [inline webforces/server/mongodbworker.py:117]: I think it is an extra condtition.

`except Exception as e:` catches the fails of the pymongo methods.

File: webforces/server/interface/dbworker.py [hunk 2]
```diff
@@ -21,6 +21,10 @@ def dropAll(self) -> None:
     def addUser(self, user) -> Tuple[DBStatus, User]:
         pass
 
+    @abc.abstractmethod
+    def updUser(self, user) -> Tuple[DBStatus, User]:
```
Comment (akorobeinikov) [inline webforces/server/interface/dbworker.py]: return only status

Comment (aobolensk) [reply webforces/server/interface/dbworker.py]: Thank you. Fixed

File: webforces/server/mongodbworker.py (hunk 1 above)
Comment (aobolensk) [reply webforces/server/mongodbworker.py:117]: `if user_d is None` handles the case when there is no exception raised.

**Review by megarungle — APPROVED** [review]

**Review by akorobeinikov — APPROVED** [review]

**Pull request merged**